modules/builtin/src/content-types/audio.js

```diff
@@ -41,14 +41,17 @@ module.exports = {
       return
     }
 
-    const link = utils.formatURL(formData.BOT_URL, formData.audio)
+    let audioUrl = formData.audio
+    if (utils.isBpUrl(audioUrl)) {
+      audioUrl = `/${audioUrl}`
+    }
+    const link = utils.formatURL(formData.BOT_URL, audioUrl)
```

Comment:
This logic is repeated multiple times. Let's just use a single method:
```js
// this way we can handle /subpath/api, api/v1, /api/v1, /some/path/api/v1
const bpUrlRegex = /api\/.*\/bots\/.*\/media\/.*/
const removeDoubleSlash = /(/([^:]\/)\/+/g

const prepareUrl = (url, baseUrl){
  if(url.startsWith('http')){
    return url
  }
  
  if(bpUrlRegex.test(url)){
    // this regex allows us to avoid having to check if the external url ends with a slash or if the url starts with one
    return `${baseUrl}/${url}`.replace(removeDoubleSlash, "$1")
  }

  return url
}

// then we can use simply `utils.prepareUrl(formData.audio, formData.BOT_URL)` in each places where we handle url
```